chart/README.md

```diff
@@ -40,9 +40,9 @@ cluster using the [Helm](https://helm.sh) package manager.
 * Supported AWS executors with AWS provider version ``8.21.0+``:
    * ``airflow.providers.amazon.aws.executors.batch.AwsBatchExecutor``
    * ``airflow.providers.amazon.aws.executors.ecs.AwsEcsExecutor``
-* Supported Airflow version: ``1.10+``, ``2.0+``
```

Comment:
The chart does actually support 1.10 though. We can't drop it until we do a version 2 of the chart.

Reply:
Okay ,so since we do not support any more airflow v1 and the helm chart is not tested against it , maybe it's time to do a v2 ?

Reply:
I was planning to do 2 closer to Airflow 3. There are a number of things I think we should adjust when we can do breaking changes.

Reply:
okay but why wait for airflow 3 for a breaking change in the helm chart ?